src/server.js

```diff
@@ -76,6 +86,8 @@ const main = async () => {
   app.post('/api/event', promiseHandler(withAuth(async req => {
     const { event, properties } = req.body
 
+    validateInput(req.body, Joi.object().keys({ event: eventSchema, properties: propertiesSchema }))
```

Comment:
That sounds correct: https://hapi.dev/family/joi/api/?v=17.1.0
In the spirit of going with the least powerful abstraction it may be best to remove the extra .keys at the moment